libdebug/ptrace/ptrace_interface.py

```diff
@@ -350,6 +350,55 @@ def finish(self: PtraceInterface, thread: ThreadContext, heuristic: str) -> None
         else:
             raise ValueError(f"Unimplemented heuristic {heuristic}")
 
+    def next(self: PtraceInterface, thread: ThreadContext) -> None:
+        """Executes the next instruction of the process. If the instruction is a call, the debugger will continue until the function returns.
+        """
+
+        # Check if the current instruction is a call and its skip
+        is_call = self.lib_trace.is_call_instruction(
+            self._global_state,
+            thread.thread_id
+        )
+
+        if is_call:
+            skip_address = self.lib_trace.compute_call_skip(
```

Comment:
Same comment as for `is_call_instruction`

Reply:
Resolved in commit [970765d](https://github.com/libdebug/libdebug/commit/970765d9fb761638831e422cc9cb8a850ce8d525)